src/Test/Utilities/Portable/Traits/CompilerFeature.cs

```diff
@@ -35,5 +35,6 @@ public enum CompilerFeature
         AsyncStreams,
         NullableReferenceTypes,
         DefaultInterfaceImplementation,
+        Attributes,
```

Comment:
>Attributes [](start = 8, length = 10)

It would be better to be more specific with the name.  We've had attributes for a long time.

Reply:
do you mean "StatementAttributes"?